test/functional/scan/test-scan.hpp

```diff
@@ -0,0 +1,84 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_SCAN_HPP__
+#define __TEST_SCAN_HPP__
+
+#include "RAJA/RAJA.hpp"
+#include "gtest/gtest.h"
+
+#include "camp/list.hpp"
+
+using namespace camp;
+using camp::list;
+using namespace camp::resources;
+
+// Unroll types for gtest testing::Types
+template<class T>
+struct Test;
+
+template<class ...T>
+struct Test<list<T...>>{
+  using Types = ::testing::Types<T...>;
+};
+
+
+// Scan functional test class
+template<typename T>
+class ScanFunctionalTest: public ::testing::Test {};
+
+
+// Define scan operation types
+using OpTypes = list< RAJA::operators::plus<int>,
+#if 0  // Parallel tests with plus operator and float data do not work
+       // likely due to precision being to low
```

Comment:
typo

Reply:
to -> too fixed.  See what I get for adding comments.... :-)